dashboard/app/models/levels/blockly.rb

```diff
@@ -203,15 +213,26 @@ def self.convert_toolbox_to_category(xml_string)
         xml.child << category_node
         block.remove
       else
+        block.remove_attribute('x')
+        block.remove_attribute('y')
         block.remove
         category_node << block
       end
     end
     default_category.remove if default_category.element_children.empty?
-    xml.serialize(save_with: XML_OPTIONS).delete("\n").strip
+    xml_string = xml.serialize(save_with: XML_OPTIONS).delete("\n").strip
+    if is_google_blockly
+      xml_string["<xml>"] = GOOGLE_BLOCKLY_NAMESPACE_XML
```

Comment:
Just curious why we think this is necessary? I think it's more descriptive, but Blockly also doesn't expect it to be there in order to do things like load blocks onto a workspace. (If they did, all of our older project/level sources that started with `<xml>` would break when we migrated to Google Blockly.)

Reply:
I was curious about this as well - see [this commit](https://github.com/code-dot-org/code-dot-org/pull/52155/commits/3123d6415ebd42516030c477a54a8a00095ac628).
I had asked @maribethb what she thought and she said that they don't necessarily rely on it now, but if they did, then the resulting problem would be really difficult to debug. The Google Blockly team doesn't anticipate that folks are manually changing the XML so it's impossible to say that they won't change this in the future. But she also said it's probably safe to remove, but not a good sign that you need to. Since we don't need to remove but temporarily, I opted to add back.